docs/config/README.md

```diff
@@ -1302,7 +1302,7 @@ heuristic = false
 
 ## Elixir
 
-The `elixir` module shows the currently installed version of [Elixir](https://elixir-lang.org/) and [Erlang/OTP](https://erlang.org/doc/).
+The `elixir` module shows the currently installed version of [Elixir](https://elixir-lang.org/) and [Erlang/OTP](https://erlang.org/doc/) if `$otp_version` variable is set on format.
```

Comment:
Please remove this change and just update the default format string (in the table!).

Reply:
Update the default format to remove `$otp_version` or are you suggesting something else?

Reply:
I'm not completely, but your PR description read like you were going to do that.

Reply:
Yeah, I was initially going to remove `$otp_version` and was playing with it but might change my mind.
That's why I put it in draft mode, so wasn't expecting to get feedback yet.

Reply:
Not changing the default format for now.